src/http.rs

```diff
@@ -123,6 +124,23 @@ where
                 })
                 .context(CallRequestSnafu)?;
 
+            let (parts, body) = response.into_parts();
```

Comment:
Note that this is all streamed in parallel with the existing data path. The `body` in this case is an asynchronous stream, and so none of the data is loaded unless the requester looks at it, and further would only be buffered temporarily up to the (currently tiny) response limit. Still, you are right, it would be good to gate this on the log level being debug or lower, which can easily be done by just setting the buffer to `None` otherwise.

Reply:
I'm also thinking this should be lowered to `trace` level logging given the verbosity of these.